department-of-veterans-affairs/abd-vro
# #2530 Enable SASL for bie kafka

## What was the problem?

The VRO Certs to access Kafka are expiring and we need to decide whether VRO will provision their own Certs and provide the Public Key to BIP Solutions to renew the ACL access or attempt to complete the RBAC migration making the Certs a non-issue.

Associated tickets or Slack threads:
- #2530 
- https://dsva.slack.com/archives/C04U8GESRT7/p1706727018289089

## How does this fix it?[^1]
This enables SASL_SSL for bie-kafka comms
## How to test this PR
- run source ./scripts/kafka-service.sh
- Watch the connection between mock-bie-kafka and svc-bie-kafka


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
